packages/core/src/routes/connector/index.ts

```diff
@@ -251,6 +264,8 @@ export default function connectorRoutes<T extends AuthedRouter>(
       body: Connectors.createGuard
         .pick({ config: true, metadata: true, syncProfile: true })
         .partial(),
+      response: connectorResponseGuard,
+      status: [200, 400, 404, 422],
```

Comment:
Do we throw 400 in the code?

Reply:
Yes, there are several "assetThat", they will throw 400 by default.